src/libs/ReportUtils.js

```diff
@@ -1437,6 +1483,27 @@ function getReportPreviewMessage(report, reportAction = {}) {
         return `approved ${formattedAmount}`;
     }
 
+    /* === Start - Check for Report Scan === */
+
+    if (shouldConsiderReceiptBeingScanned) {
+        const filteredIouReportActions = getSortedMoneyRequestActions(report.reportID);
+        const lastIouReportAction = _.first(filteredIouReportActions);
+        const lastIouReportActionTransaction = TransactionUtils.getLinkedTransaction(lastIouReportAction);
+
+        // get lastTransaction
+        // eslint-disable-next-line no-use-before-define
+        const transactionsWithReceipts = getSortedTransactionsWithReceipts(report.reportID);
+        const lastTransaction = _.first(transactionsWithReceipts);
+
+        const transactionIsLastReportAction = _.isEqual(lastIouReportActionTransaction, lastTransaction);
+
+        if (lastTransaction && transactionIsLastReportAction && TransactionUtils.isReceiptBeingScanned(lastTransaction)) {
+            return Localize.translateLocal('iou.receiptScanning');
+        }
+    }
```

Comment:
For workspace Chat LHN subtitle. Same function is used for that and Expense report view

Reply:
Why not, let me change it

Reply:
@allroundexperts Since `getLastMessageTextForReport` passes the `iouReport` to `getReportPreviewMessage` even when its Workspace chat, do you think the inital approach was better? src/libs/OptionsListUtils.js 
![image](https://github.com/Expensify/App/assets/65986357/338c4d83-0a59-42e3-baa9-a5955891f466)